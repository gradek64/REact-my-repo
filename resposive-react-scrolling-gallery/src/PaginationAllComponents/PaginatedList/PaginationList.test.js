import React from 'react';
import { shallow } from 'enzyme';
import PaginationList from './index';

const props = {
  startFrom: 0,
  paginateBy: 4,
  list: [
    {
      _id: '593e9297e17df20c4a237d42',
      index: 0,
      price: 937180,
      picture: 'images/tn_property01.jpg',
      city: 'Singer',
      address: '914 Argyle Road Alley',
      latitude: -33.944576,
      longitude: 151.25584,
      bedrooms: 2,
      bathrooms: 2,
      carSpaces: 2,
    },
    {
      _id: '593e9297ec4cca9c56bf61af',
      index: 1,
      price: 703725,
      picture: 'images/tn_property02.jpg',
      city: 'Machias',
      address: '255 Raleigh Place',
      latitude: -33.944471,
      longitude: 151.2541,
      bedrooms: 2,
      bathrooms: 1,
      carSpaces: 0,
    }],
  children: {},
};

test('renders correctly', () => {
  const wrapper = shallow(<PaginationList {...props} />);
  expect(wrapper).toMatchSnapshot();
});
