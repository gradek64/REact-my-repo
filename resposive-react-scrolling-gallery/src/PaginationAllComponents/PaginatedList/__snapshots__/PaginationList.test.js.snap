// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <Component
    list={
      Array [
        Object {
          "_id": "593e9297e17df20c4a237d42",
          "address": "914 Argyle Road Alley",
          "bathrooms": 2,
          "bedrooms": 2,
          "carSpaces": 2,
          "city": "Singer",
          "index": 0,
          "latitude": -33.944576,
          "longitude": 151.25584,
          "picture": "images/tn_property01.jpg",
          "price": 937180,
        },
        Object {
          "_id": "593e9297ec4cca9c56bf61af",
          "address": "255 Raleigh Place",
          "bathrooms": 1,
          "bedrooms": 2,
          "carSpaces": 0,
          "city": "Machias",
          "index": 1,
          "latitude": -33.944471,
          "longitude": 151.2541,
          "picture": "images/tn_property02.jpg",
          "price": 703725,
        },
      ]
    }
  />
</Fragment>
`;
